tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -155,9 +166,6 @@ if(WIN32)
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /D_ITERATOR_DEBUG_LEVEL=0")
   set(CMAKE_CXX_FLAGS_MINSIZEREL "${CMAKE_CXX_FLAGS_MINSIZEREL} /D_ITERATOR_DEBUG_LEVEL=0")
   set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /D_ITERATOR_DEBUG_LEVEL=0")
-
-  # Try to avoid flaky failures due to failed generation of generate.stamp files.
-  set(CMAKE_SUPPRESS_REGENERATION ON)
```

Comment:
Please add these lines back, because they are necessary to make the build more robust on some platforms.